pom.xml

```diff
@@ -111,6 +111,10 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>apache-httpcomponents-client-4-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>pipeline-stage-step</artifactId>
```

Comment:
It may make more sense to implement the `when` step in the [pipeline-stage-step-plugin](https://github.com/jenkinsci/pipeline-stage-step-plugin)?

Reply:
Definitely this dependency may not be here.